### PR TITLE
Add withdraw reward to cosmos

### LINKED ISSUE #224: [ COSMOS ] implement withdrawing rewards functionality

### PR DESCRIPTION
**Pull request by leoneparise**
I was able to withdraw but It never confirm the transaction on unstake. @dpereskokov can you take a look?

Fixes #224

### CODE REVIEW

**Comment by dpereskokov** [comment]
I've just tried to unstake and it confirms transaction showing success message. Or you mean it doesn't confirm withdrawing rewards transaction?

File: src/app/coins/coin-provider-config.ts [hunk 1]
```diff
@@ -64,4 +64,4 @@ export class UnitConverter<T extends CoinProviderConfig> {
 // CONSTS
 export const BALANCE_REFRESH_INTERVAL = 60000;
 export const STAKE_REFRESH_INTERVAL = 115000;
-export const TX_WAIT_CHECK_INTERVAL = 2 * 1000;
+export const TX_WAIT_CHECK_INTERVAL = 5 * 1000;
```
Comment (dpereskokov) [inline src/app/coins/coin-provider-config.ts]: Why did you decide to increase checking interval?

Comment (leoneparise) [reply src/app/coins/coin-provider-config.ts]: In creased it because I started to receive some errors due to making a call every 2 seconds.

**Comment by leoneparise** [comment]
> @leoneparise I've just tried to unstake and it confirms transaction showing success message. Or you mean it doesn't confirm withdrawing rewards transaction?

The message was not showing to me. Really strange.

**Comment by dpereskokov** [comment]
let me check on this branch

**Pull request merged**